### DISCUSSION TITLE
How to make multiple blogs

### DISCUSSION THREAD

**Discussion by lkh23333** [alshedivat/al-folio#2295] [Q&A] [answered]
Currently I modify the submenu in the template to travel, films, research and miscellaneous. I would like to separately create a blog for each one of them. I have made some attempts but they always navigate to the same original blog page, even if I have deleted blog.md. Not sure what happens inside the workflow. Any easy ways for me to do this? Thank you in advance!

**Comment by george-gca** [comment]
You can probably use [categories](https://jekyllrb.com/docs/posts/#categories) for that, and filtering only the posts that have some specific category when entering that section. A [quick google search](https://www.google.com/search?q=jekyll+multiple+blogs&oq=jekyll+multiple+blogs&gs_lcrp=EgZjaHJvbWUyBggAEEUYOdIBCDQxNzFqMGoxqAIAsAIA&ie=UTF-8) actually returned a lot of possibilities.

**Reply by lkh23333** [reply]
Thank you for your answer! Unfortunately, no matter how I modify according to those tutorials from Googling, all generated blog stuff still goes to the blog folder, instead of the folder I create (e.g., blogs_travel). There should be a lot of hidden settings in the codes designating this behavior, but I have no idea where to start from.

**Reply by george-gca** [reply]
What have you tried/what worked already? The answers on [stackoverflow](https://stackoverflow.com/a/14583346/14459430) seem pretty self-explanatory, given you have played a little with the template and the jekyll docs, and understand the directory structure.

**Reply by george-gca** [reply]
The easiest solution is to simply categorize your blog posts, and then display only the posts with the category you want. About the path for the blogs, you can use the `permalink` variable in the frontmatter to define it.

**Reply by lkh23333** [reply, marked as answer]
Thank you very much! I have solved the problem by setting the category and permalink! Also I disabled jekyll-archives so that the generated blog folder is clean now.

**Reply by DamonsJ** [reply]
@george-gca 
![image](https://github.com/user-attachments/assets/5d9c0463-c095-4b60-b9a3-16935b565ab6)

I want to make "Programming" and "Vision" both blog type.
I mean "Programming" and "Vision"  both have different tags and different names and event different folders

Can I do it the way you said?

Thanks！

**Reply by george-gca** [reply]
For this, you might want to change `jekyll-archives` for [jekyll-archives-v2](https://github.com/george-gca/jekyll-archives-v2). It is a fork of the former with support for multiple collections. I was planning to change it in the template in a later version, but maybe I'll add it before that.

**Reply by george-gca** [reply]
You can see an example of its usage in #2866.

**Reply by DamonsJ** [reply]
Thanks so much!

**Reply by DamonsJ** [reply]
@george-gca  sorry to bother!

I forked [multi-language-al-folio](https://github.com/george-gca/multi-language-al-folio) and create another blog named programming

but I got same content with blog, can u help me please?
here are the steps I have taken.
and the code also in here.

https://github.com/george-gca/multi-language-al-folio/discussions/46

Thanks!

**Reply by george-gca** [reply]
One thing to note is that you should not replace `posts` everywhere blindly. Your issue is probably because how `jekyll-paginate-v2` works. See [this example](https://github.com/sverrirs/jekyll-paginate-v2/blob/master/examples/03-tags/) in their repo.

Even though the collection is set [here](https://github.com/sverrirs/jekyll-paginate-v2/blob/7bc2634c04c49120b674cc3506275f85c4389f5c/examples/03-tags/paginations/romance-historical.md?plain=1#L7) as `romance`, the use in the paginator is still as `paginator.posts`, as can be seen [here](https://github.com/sverrirs/jekyll-paginate-v2/blob/7bc2634c04c49120b674cc3506275f85c4389f5c/examples/03-tags/_layouts/home.html#L22).
